.github/release-drafter.yml

```diff
@@ -9,11 +9,22 @@ categories:
       - 'bugfix'
       - 'bug'
   - title: '🧰 Maintenance'
-    label: 'chore'
+    label: 
+      - 'chore'
+      - 'dependencies'
```

Comment:
i think `dependencies` would fit under `chore`

Reply:
It might be good to mark certain generated PRs with dependencies, we can identify them with that label.

Reply:
I second Mike